### ISSUE TITLE
[🐛BUG] 对timestamp进行embedding得到的结果过大

### ISSUE THREAD

**Issue by FineArtz** [RUCAIBox/RecBole#1062]
在使用Wide&Deep模型和tmall-click数据集时，train loss完全不变。之后我发现在模型计算float_fields_embedding时，
```
# Line 1052, model/layers.py
float_fields_embedding = self.embed_float_fields(float_fields, embed=True)
```
由于timestamp的类型是float且数字巨大，得到的embedding结果都在1e8以上，从而在计算总的embedding时
```
# Line 1077, model/layers.py
return torch.sum(torch.cat(total_fields_embedding, dim=1), dim=1) + self.bias  # [batch_size, output_dim]
```
timestamp的embedding完全覆盖了其他feature的embedding，导致模型无法正常训练。

我不太清楚是不是我对代码理解的有问题，或者我的config设置不太对，希望您能解答，谢谢！

**Comment by chenyuwuxin** [comment]
@FineArtz 感谢您的关注！
一般我们不对timestamp进行embedding，而常将其作为过滤或者排序的标准。如果将timestamp作为dense类型的feature，确实会导致最终的模型输出数值过大等问题。对于时间这种类型的数据，您可以先将其转化为Year / Month / Day 等数值范围较小的特征进行embedding，或者作为离散类型的**token**特征。

**Comment by FineArtz** [comment]
我觉得我发现问题在哪里了。文档中写，`normalize_all`参数默认值为`True`，所以我没有在自己的config中指定这一参数的值。但我仔细查看代码后发现，如果数据集不是默认的`ml-100k`，该参数的默认值其实为`None`，导致float_fields没有正则化。

我现在理解的参数初始化流程和涉及到的文件中，只有`properties/dataset/ml-100k.yaml`中`normalize_all`被设置为`True`，其他默认参数文件均没有指定该参数，或指定为`~`。不知道我的理解是否正确？

**Comment by chenyuwuxin** [comment]
@FineArtz 您的理解是正确的，目前只有ml-100k的配置文件设置了`normalize_all`为True，而默认的`normalize_all`参数为`~`。为了对float类型数据进行标准化，你可以在配置文件中设置：`normalize_all: True`。

**Comment by FineArtz** [comment]
感谢！希望文档 https://recbole.io/docs/user_guide/config/data_settings.html#preprocessing 的这一部分的参数说明也能更新。
